### DISCUSSION TITLE
Should we allow changing namespace and svcaccount for scan jobs?

### DISCUSSION THREAD

**Discussion by nilesh-akhade** [aquasecurity/starboard#909] [Development]
Hey,
The starboard code is very well structured and documented.

We found some things which can be made configurable. Will it be considered an improvement?

Actually, we wanted to execute scan jobs in a different namespace. So we made the following changes.
https://github.com/aquasecurity/starboard/compare/main...catalogicsoftware:main

I am new here and excited to contribute. Feel free to suggest any issue to work on :)

Thanks

**Comment by danielpacak** [comment]
Thank you for the feedback @nilesh-akhade I think we have two possibly similar requirements to yours already captured in #817 and #646.
Could you provide your use cases and explain why making namespace and service account configurable would help?

**Reply by nilesh-akhade** [reply]
We are introducing a way to run vulnerability scans on the clusters we back up. We are using Starboard as the scanning framework to identify vulnerabilities on K8s clusters.
Since we are integrating by importing starboard as a dependency. We need more control over the namespace and svc account under which the scan job runs.

**Reply by nilesh-akhade** [reply]
Issues #817 and #646 will also be covered partially by this change. Right?
